setup.py

```diff
@@ -55,7 +61,8 @@
               "sphinx-autodoc-typehints",
               "flake8",
               "myst-parser",
-          ]
+          ],
+          "processing": processing_require
```

Comment:
I think "local" should be in this "extra" name. Just "processing" seems to suggest that a default install without extras does not allow processing.
e.g. "localprocessing", or even just "local"

Reply:
yes, I was also thinking about `localprocessing`, but then it seemed too long. But it makes more sense since it is self explanatory. I will rename it into `localprocessing`.